components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -140,7 +146,7 @@ public ActionExecutionStatus execute(ActionType actionType, String[] actionIdLis
         } catch (ActionExecutionRuntimeException e) {
             DIAGNOSTIC_LOGGER.logSkippedActionExecution(actionType);
             LOG.debug("Skip executing actions for action type: " + actionType.name(), e);
-            return new ActionExecutionStatus(ActionExecutionStatus.Status.FAILED, eventContext);
+            return new FailedStatus();
```

Comment:
This should hold the same contract as above. why do you think it should be failed ?

Reply:
This is fixed with 148073eab6305adc3e26c71ca755a0550e805f54